common/smoke-tests/pom.xml

```diff
@@ -124,7 +124,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-identity</artifactId>
-      <version>1.4.4</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
+      <version>1.4.5</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
```

Comment:
It's already 1.4.5 in `feature/azure-spring-cloud-4.0` branch.
Could you please remove this commit in this PR?